stdlib/testing/usage/reads.flux

```diff
@@ -0,0 +1,3219 @@
+package usage_test
+
+import "testing"
+
+// This dataset has been generated with this query:
+// from(bucket: "system_usage")
+//     |> range(start: 2019-08-01T12:00:00Z, stop: 2019-08-01T14:00:00Z)
+//     |> filter(fn: (r) =>
+//         r._measurement == "http_request"
+//         and r._field == "resp_bytes"
+//         and r.endpoint == "/api/v2/query"
+//         and (r.status == "200" or r.status == "500" or r.status == "400")
```

Comment:
I wonder why you needed to filter on `r.status` like this?  Seems like those are the only status codes that queryd can generate.

Reply:
👍 I guess you are right. I could retry and comment out that line. Anyways, the result would be the same (in the case they are the only error codes that queryd can generate).

For the sake of curiosity, `401` and `403` are not possible?

Reply:
No, actually you're right. I think you can get 401 and 403 from `queryd` now, since we now have a `flux.Error` type that can produce them.  Eg., if you try to read from a bucket you don't have read permissions for, you would get 401 or 403.